eventActions/accountabilityActions.js

```diff
@@ -74,17 +98,43 @@ class accountabilityActions {
 			}
 		}
 	}
-
-	// Unpin message by removing the reaction
-	static async userManuallyUnpinsMessage(reaction, user){
-		var curMessage = reaction.message;
-		if(curMessage.channel.id === config.channels.accountability){
-			if(curMessage.author.id === user.id){
-				if(curMessage.pinned){
-					curMessage.unpin();
-					curMessage.channel.send('Hey, ' + user.username + ', I\'ve unpinned your message as requested!');
-				}
+	// Add a random reaction to a message sent
+	static async addReaction(client, message){
+		if(message.channel.id != config.channels.accountability) return;
+		if(message.content.toLowerCase().includes('!unpin')) return;
+		// Define an array of emojis to pull from
+		var random_emotes = ['💯', '529488404103299074', '👍', '🔥', '🙌', '👏', '👌', '💪', '587455770283737110', '531663637694382089'];
+
+		// Flag emotes
+		var length = random_emotes.length;
+		var flags = [
+			{ language: 'french', emote: '🇫🇷'},
+			{ language: 'spanish', emote: '🇪🇸'},
+			{ language: 'italian', emote: '🇮🇹'}
+		];
+
+		// Define special emotes (I didn't want to put all of them in the configuration...)
+		var checkmark = '562072692086276118';
```

Comment:
I think this is already in config, they should generally be there, so it's easy to change them in the future.

Reply:
The checkmark defined in configuration is the normal ✅ but I'll add "pom" and "yes2" as options in the config for these ones.

Reply:
Variables stored in configuration, pom and yes2 added to config